### ISSUE TITLE
Moore_2019_2: Processing submission

### ISSUE THREAD

**Issue by CaitlanB** [traitecoevo/austraits.build#250]
**Source**: Moore et al (2018) Effects of drought and fire on resprouting capacity of 52
temperate Australian perennial native grasses *New Phytologist* 10.1111/nph.15480 *Custodian**: James Camac
**Handled by**: ?Austraits team member?

- [x] Contact made with contributor (label as `waiting on response` if waiting for files)
- [x] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [x] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [x] Metadata complete, builds into Austraits
- [x] Report reviewed by AusTraits team
- [x] Report sent to contributor ( label as `waiting on response` )
- [x] Response received from contributor
- [x] Files updated & report finalised
- [x] Final checks ( all tests pass )
- [x] Branch merged into master branch (issue `closed` )

NB: 

- if you need to change the config files, e.g. to add a unit conversion or define a new trait. This should be done on the master branch (via pull or rebase)
